### PR TITLE
fix: test service spec error detected by the validator

### PR DESCRIPTION
**Pull request by jjtang1985**
The validation step was skipped. I fixed one spec error detected by the validator.

### CODE REVIEW

**Review by FrankEssenberger — DISMISSED** [review]
LGTM. Only one thing you could do - check if the generator fails with exit code != 0 in case of a faulty spec file.

**Comment by FrankEssenberger** [comment]
Only one thing you could do - check if the generator fails with exit code != 0 in case of a faulty spec file.

**Comment by jjtang1985** [comment]
@FrankEssenberger , I skipped the validation again and only kept the fix for the test service spec after discussed with @marikaner , because this might be related to a decision about whether we allow duplicate `operationId` in the spec file.

File: test-resources/openapi-service-specs/swagger-yaml-service.yml [hunk 1]
```diff
@@ -41,7 +41,8 @@ paths:
       responses:
         '200':
           description: Success
-          type: string
+          schema:
+            type: string
```
Comment (jjtang1985) [inline test-resources/openapi-service-specs/swagger-yaml-service.yml:45]: the real fix

Comment (marikaner) [reply test-resources/openapi-service-specs/swagger-yaml-service.yml:45]: Good catch, I though this was 2.0 syntax, but I crosschecked the docs and it isn't ;)

**Review by marikaner — APPROVED** [review]
LGTM

**Pull request merged**